tools/buildtools/topologyeditors/macros.xml

```diff
@@ -0,0 +1,14 @@
+<macros>
+    <token name="@TOOL_VERSION@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="3.8">python</requirement>
+            <yield />
+        </requirements>
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.1016/j.softx.2015.06.001</citation>
```

Comment:
here is no yield, so you can not expand it and add new citations in your tool, but you are trying to do it